ui/src/apps/clusterinfo/RootComponent.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from 'react';
+import { Row, Col } from 'antd';
+import { HashRouter as Router } from 'react-router-dom';
+
+import { Component_panel, ComponentPanel, MonitorAlertBar } from './components';
```

Comment:
`Component_panel` is a bad name, the component should use a double camel naming style.

Reply:
Changed